File: src/001.Array/448.FindAllNumbersDisappearedInAnArray/448.找到所有数组中消失的数字.js
/*
 * @lc app=leetcode.cn id=448 lang=javascript
 *
 * [448] 找到所有数组中消失的数字
 */

// @lc code=start
/**
 * @param {number[]} nums
 * @return {number[]}
 */
var findDisappearedNumbers = function (nums) {
	let lackNums = [];
	for (var i = 1; i <= nums.length; i++) {
		if (!nums.includes(i)) {
			lackNums.push(i);
		}
	}
	return lackNums;
};
// @lc code=end

File: src/001.Array/448.FindAllNumbersDisappearedInAnArray/448.找到所有数组中消失的数字.go
/*
 * @lc app=leetcode.cn id=448 lang=golang
 *
 * [448] 找到所有数组中消失的数字
 */

// @lc code=start
func findDisappearedNumbers(nums []int) (result []int) {
	for _, v := range nums {
		index := (v - 1) % len(nums)
		nums[index] += len(nums)
	}
	for i, v := range nums {
		if v <= len(nums) {
			result = append(result, i+1)
		}
	}
	return
}

// @lc code=end

